massgov/mayflower
# DP-9571 Remove raw filter from _patterns/01-atoms/11-text/raw-html.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from _patterns/01-atoms/11-text/raw-html.twig.

Use of the template in Mayflower:
```
 ~/MassIT/mayflower/ [feature/DP-9571_atoms-text-raw-html] grep -r '11-text/raw-html.twig' patternlab/styleguide/source
patternlab/styleguide/source/_patterns/05-pages/organization-detail.json:          "path": "@atoms/11-text/raw-html.twig",
patternlab/styleguide/source/_patterns/05-pages/organization-detail.json:          "path": "@atoms/11-text/raw-html.twig",
patternlab/styleguide/source/_patterns/05-pages/organization-detail.json:          "path": "@atoms/11-text/raw-html.twig",
 ~/MassIT/mayflower/ [feature/DP-9571_atoms-text-raw-html]
```

Note:  The template is used in multiple areas in Drupal.

## Related Issue / Ticket

- [DP-9571](https://jira.mass.gov/browse/DP-9571)

## Steps to Test


1. In Mayflower, go to organization details page and check for any display issue.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
